src/extension/templates/languages.ts

```diff
@@ -200,13 +191,22 @@ const languages: { [code: string]: Language } = {
         problemMatcher: [],
       },
       {
-        label: "deploy-smartcontract",
+        label: "transfer-gas-to-wallets",
         dependsOnLabel: "create-wallet-bob",
         group: "set-private-chain",
         type: "shell",
         command: "neoxp",
         args: ["batch", "-i", "test/$_CONTRACTNAME_$Tests.neo-express", "test/setup-test-chain.batch"],
         problemMatcher: [],
+      },
+      {
+        label: "build",
+        dependsOnLabel: "transfer-gas-to-wallets",
+        group: "build",
+        command: "neo3-boa",
+        type: "shell",
+        args: ["src/$_CONTRACTNAME_$_contract.py"],
+        problemMatcher: [],
         autoRun: true,
```

Comment:
Moved the build task to the bottom, because, if the user doesn't have `neo3-boa` installed all the other tasks would never start.